sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -113,14 +113,6 @@ export interface DeleteConfigurationSettingOptions extends HttpOnlyIfUnchangedFi
 export interface DeleteConfigurationSettingResponse extends SyncTokenHeaderField, HttpResponseFields, HttpResponseField<SyncTokenHeaderField> {
 }
 
-// @public
-export interface ErrorDetail {
```

Comment:
Do you know when this was first exposed?

Reply:
It might have been during the release of snapshot since it was exported with the LRO work